src/applications/ezr/tests/unit/components/FormAlerts/IdentityVerificationAlert.unit.spec.js

```diff
@@ -18,7 +18,7 @@ describe('ezr <IdentityVerificationAlert>', () => {
       const { container } = render(<IdentityVerificationAlert />);
       const selector = container.querySelector('.vads-c-action-link--green');
       expect(selector).to.exist;
-      expect(selector).to.have.attr('href', '/verify');
+      expect(selector).to.have.attr('href', APP_URLS.verify);
```

Comment:
You may end up having to pull this assertion out. The URLs are loaded in from webpack, and thus don't seem to be available for unit test purposes.

Reply:
FWIW, the test runs green for me both in isolation and as part of an overall `yarn test:unit`.

Reply:
@Mitch-A6 , for sure. I checked it after I commented... if you log out the DOM in the unit test, the href is blank because of the `getAppUrl` function going to webpack for the URLs. So this assertion actually works because the empty href equals the empty value from the constant.

Reply:
Oof, the misleading green is rough.... Would it make sense to test this by stubbing out the return value and confirming that it comes through?